pkg/ttnpb/field_mask_validation.go

```diff
@@ -620,6 +620,7 @@ var AllowedFieldMaskPathsForRPC = map[string][]string{
 	// Users:
 	"/ttn.lorawan.v3.UserRegistry/Get":                 omitFields(UserFieldPathsNested, "password", "temporary_password"),
 	"/ttn.lorawan.v3.UserRegistry/Update":              omitFields(UserFieldPathsNested, "password", "password_updated_at"),
+	"/ttn.lorawan.v3.UserRegistry/List":                omitFields(UserFieldPathsNested, "password", "temporary_password"),
```

Comment:
Use ascending sort

Reply:
We typically order Create - Get - List - Update - Delete - Search (CRUDS), but I'm not sure if we do that consistently.